frontend/workflows/sourcecontrol/package.json

```diff
@@ -12,8 +12,8 @@
     "build": "yarn clean && yarn compile",
     "clean": "rm -rf ./dist && rm -f tsconfig.tsbuildinfo",
     "compile": "tsc -b",
-    "compile:dev": "tsc --sourceMap",
-    "compile:watch": "BABEL_ENV=build babel src --out-dir dist --source-maps --extensions .js,.jsx,.ts,.tsx --ignore **/tests --watch",
+    "compile:dev": "esbuild --target=es2019 --outdir=dist --sourcemap src/*.tsx",
+    "compile:watch": "esbuild --target=es2019 --outdir=dist --sourcemap --watch src/*.tsx",
```

Comment:
why limit to 2019?

Reply:
This is to enable support for optional chaining

Reply:
>The default is esnext which should cover all the latest JS features. I
believe it targets es2020 for now.

Huh I swear I tried this and it didn't work over the weekend but magically it does now 🤔

Reply:
hmm i don't get why that would happen, null coalesce is part of ES2020

Reply:
yeah, I didn't dig into the project enough to figure out why this is an issue but it's definitely not transpiling it